core/kazoo_endpoint/src/kz_endpoint.erl

```diff
@@ -1562,6 +1563,22 @@ bowout_settings('false') ->
     ,{<<"Loopback-Bowout">>, <<"true">>}
     ].
 
+
+-spec maybe_set_call_forward_fail(ccv_acc()) -> ccv_acc().
+maybe_set_call_forward_fail({_Endpoint, _Call, 'undefined', _CCVs}=Acc) ->
+    Acc;
+maybe_set_call_forward_fail({Endpoint, Call, CallFwd, CCVs}=Acc) ->
+    case kapps_config:get_is_true(?CONFIG_CAT, <<"fail_on_single_call_forward_reject">>, 'false') of
+        'true' ->
+            Fail = kapps_config:get_binary(?CONFIG_CAT, <<"fail_on_reject_val">>, <<"NORMAL_CLEARING">>),
```

Comment:
@icehess thanks for the help, i have tried continue_on_fail too. after some digging it seems that, when fail_on_single_reject is set to something it will set that value with three more causes which are `NORMAL_CLEARNING` `USER_BUSY` and `NO_ANSWER`